### PR TITLE
Action Remote Shims Register/Unregister

### PR DESCRIPTION
**Pull request by ettec**
support for executing an action remotely (executable capability more generally) as part of -> https://smartcontract-it.atlassian.net/browse/CAPPL-108

### CODE REVIEW

File: core/capabilities/remote/executable/server.go
Comment (cedric-cordenier) [inline core/capabilities/remote/executable/server.go]: @ettec Can we also be permissive here: if the attribute isn't at the path; then that's arguably a good thing right? We don't need to delete

**Review by cedric-cordenier — DISMISSED** [review]
I'm not 100% keen on how we are mixing concepts Executable capability vs Action/Consensus/Target etc within the same level of the API (the config we store in the capabilities registry)

However I'm not going to block the PR on this

File: core/capabilities/remote/executable/server.go
Comment (ettec) [reply core/capabilities/remote/executable/server.go]: yes, good point, changing....

**Comment by ettec** [comment]
> I'm not 100% keen on how we are mixing concepts Executable capability vs Action/Consensus/Target etc within the same level of the API (the config we store in the capabilities registry)
> 
> However I'm not going to block the PR on this

 issue raised on this to track so we can come back to later-> https://smartcontract-it.atlassian.net/browse/CAPPL-291

**Review by justinkaseman — APPROVED** [review]

**Review by agparadiso — APPROVED** [review]

**Comment by ettec** [comment]
something has gone deeply screwed up with the approval process for this PR, closing to open another in hope it fixes it

**Review by reductionista — APPROVED** [review]

**Review by vyzaldysanchez — APPROVED** [review]

**Pull request merged**
